configurators/pom.xml

```diff
@@ -197,14 +208,20 @@
         <dependency>
             <groupId>com.googlecode.java-diff-utils</groupId>
             <artifactId>diffutils</artifactId>
-            <version>1.2.1</version>
+            <version>1.3.0</version>
             <scope>test</scope>
         </dependency>
 
         <dependency>
             <groupId>org.mariadb.jdbc</groupId>
             <artifactId>mariadb-java-client</artifactId>
-            <version>2.3.0</version>
+            <version>2.7.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jasypt</groupId>
+            <artifactId>jasypt</artifactId>
+            <version>1.9.3</version>
```

Comment:
Can we use the one with the lite classifier everywhere, as now both versions are used

Reply:
yes we can, I just discovered the new tools for dependencies from Intellij yesterday. you can search for packages and add them directly in your poms